### ISSUE TITLE
Tweet.replies() doesn't contain all the replies from a tweet.

### ISSUE THREAD

**Issue by ColonialDagger** [d60/twikit#52]
Having an issue where `Tweets.replies()` doesn't get all the tweets in a tweet thread.

For example, if I pull [this tweet from @Destiny2Team](https://twitter.com/Destiny2Team/status/1766274441192046916), I only get the initial tweet I pulled as the base object and in the `tweet.replies` object, I don't get all the tweet replies in the chain.

When I get the tweet the using `user.get_tweets()` and go to that specific tweet (currently index 16), I only get the last 2 tweets in that thread for that specific entry.

![Pasted image 20240328201130](https://github.com/d60/twikit/assets/33355627/25a23972-093f-4c53-ac11-a5b3e27f0363)

```python
>>> user = client.get_user_by_screen_name('Destiny2Team')
>>> tweets = user.get_tweets('Tweets')
>>> len(tweets[16].replies)  # Should return 7
2
```

When I get the tweet using `client.get_tweet_by_id()`, I only get the last 5 tweets in that thread.

![Pasted image 20240328202742](https://github.com/d60/twikit/assets/33355627/da73efb4-24dc-4d9f-81ba-0e27ab2a3d5c)

```python
>>> tweet = client.get_tweet_by_id(1766274441192046916)
>>> len(tweet.replies)  # Should return 7
5
```

I assume that on the backend, twikit is limiting how many replies it will look for. Is it possible to have a user-defined setting where that reply limit can be raised/lowered for each implementation?

**Comment by d60** [comment]
@ColonialDagger 
The tweets obtained using `get_tweet_by_id` do not include all the replies because the API response does not contain that information. To retrieve all the replies, you need to fetch the tweet again using `get_tweet_by_id`.

```python
>>> user = client.get_user_by_screen_name('Destiny2Team')
>>> tweet_ = user.get_tweets('Tweets')[16]
>>> tweet = client.get_tweet_by_id(tweet_.id)

>>> replies = tweet.replies
>>> print(replies)
[<Tweet id="...">, <Tweet id="...">, ..., <Tweet id="...">]

>>> more_replies = replies.next()
>>> print(more_replies)
[<Tweet id="...">, <Tweet id="...">, ..., <Tweet id="...">]
```

**Comment by ColonialDagger** [comment]
Gotcha, I was able to figure it out. Thanks for the help!

For anybody who comes across this thread in the future. here's what I did:

1. Grab a user's tweets and go through them one by one
2. If there are replies to the base tweet, I fetch that specific tweet again and it will give be the children of that tweet.
3. I enter a loop constantly searching for children that share the same user ID as the parent.
4. Using my full list of children, I can fully build out my tweet thread.
